packages/filesystem/src/browser/file-upload-service.ts

```diff
@@ -254,6 +254,7 @@ export class FileUploadService {
         } catch (error) {
             uploadSemaphore.cancel();
             if (!isCancelled(error)) {
+                this.messageService.error(`An error occurred while uploading a file. ${error.toString()}`);
```

Comment:
maybe add localization for this message just with `nls.localize('key', 'message {0}', error)`

Reply:
Good idea